firebase-vscode/package.json

```diff
@@ -64,7 +64,7 @@
       "firebase": [
         {
           "type": "webview",
-          "id": "firebase.sidebarView",
+          "id": "sidebar",
```

Comment:
Any chance of collisions with other extensions creating a webview named "sidebar" or will this be scoped to the parent "firebase"?